renku/api/storage.py

```diff
@@ -31,6 +31,10 @@
 
 HAS_LFS = call(['git', 'lfs'], stdout=PIPE, stderr=STDOUT) == 0
 
+# Batch size for when renku is expanding a large list
+# of files into an argument string.
+ARGUMENT_BATCH_SIZE = 1000
```

Comment:
Wouldn't the problem still be there if we pass 1000 arguments with each size being bigger than 150 bytes? Should we make default batch size smaller?

Reply:
Changed it to 100.